src/relayclient/ContractInteractor.ts

```diff
@@ -421,6 +432,36 @@ export default class ContractInteractor {
     const hub = this.relayHubInstance
     return hub.contract.methods.addRelayWorkers(workers)
   }
+
+  /**
+   * Web3.js as of 1.2.6 (see web3-core-method::_confirmTransaction) does not allow
+   * broadcasting of a transaction without waiting for it to be mined.
+   * This method sends the RPC call directly
+   * @param signedTransaction - the raw signed transaction to broadcast
+   */
+  async broadcastTransaction (signedTransaction: PrefixedHexString): Promise<PrefixedHexString> {
+    return await new Promise((resolve, reject) => {
+      if (this.provider == null) {
+        throw new Error('provider is not set')
+      }
+      this.provider.send({
+        jsonrpc: '2.0',
+        method: 'eth_sendRawTransaction',
+        params: [
+          signedTransaction
+        ],
+        id: Date.now()
+      }, (e: Error | null, r: any) => {
+        if (e != null) {
+          reject(e)
+        } else if (r.error != null) {
+          reject(r.error)
```

Comment:
please verify whether we need to `reject(error)` or `reject( new Error(error))`

Reply:
Type declaration says `reason?: any` and I don't see a reason to wrap it again. It will be an error of rejected promise anyways. What do you think?

Reply:
types sucks I don't know if they declare the type correctly. What I do know is that I want to get the error just like we used to without . I only suggested to verify..